### PR TITLE
added logging to climber

### PR DESCRIPTION
**Pull request by dunep**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -34,6 +34,7 @@
 public final class Constants {
 
   public static class constClimber {
+    public static final NeutralModeValue PITCH_NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
```
Comment (TaylerUva) [inline src/main/java/frc/robot/Constants.java]: Does this variable name make sense?

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 2]
```diff
@@ -37,20 +42,30 @@ public void configure() {
 
     climberConfig.SoftwareLimitSwitch.ReverseSoftLimitEnable = true;
     climberConfig.SoftwareLimitSwitch.ReverseSoftLimitThreshold = climberPref.climberMotorReverseLimit.getValue();
+
+    climberConfig.MotorOutput.NeutralMode = constClimber.PITCH_NEUTRAL_MODE_VALUE;
     climberMotor.getConfigurator().apply(climberConfig);
+
+    climberMotor.setControl(climberMotorReq.withOutput(12));
```
Comment (TaylerUva) [inline src/main/java/frc/robot/subsystems/Climber.java]: What is this line doing?

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 3]
```diff
@@ -37,20 +42,30 @@ public void configure() {
...
+  }
+
   public void setNeutralOutput() {
     climberMotor.setControl(new NeutralOut());
   }
 
   @Override
   public void periodic() {
+    SmartDashboard.putNumber("Climber position(Rotations)", getPosition());
+    SmartDashboard.putString("climber motor encoder", "DutyCycle");
```
Comment (TaylerUva) [inline src/main/java/frc/robot/subsystems/Climber.java]: What is this line doing?

**Review by TaylerUva — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Climber.java (hunk 3 above)
Comment (dunep) [reply src/main/java/frc/robot/subsystems/Climber.java]: nu uh

Comment (dunep) [reply src/main/java/frc/robot/subsystems/Climber.java]: we where trying to show the control type should we have this?

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 4]
```diff
@@ -6,18 +6,23 @@
...
 
 import com.ctre.phoenix6.controls.NeutralOut;
+import com.ctre.phoenix6.controls.VoltageOut;
 
 public class Climber extends SubsystemBase {
   TalonFX climberMotor;
 
   TalonFXConfiguration climberConfig;
 
+  final VoltageOut climberMotorReq = new VoltageOut(0);
```
Comment (TaylerUva) [inline src/main/java/frc/robot/subsystems/Climber.java]: Is this line of code being used?

**Review by TaylerUva — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 5]
```diff
@@ -37,20 +42,29 @@ public void configure() {
...
+  }
+
   public void setNeutralOutput() {
     climberMotor.setControl(new NeutralOut());
   }
 
   @Override
   public void periodic() {
+    SmartDashboard.putNumber("Climber position(Rotations)", getPosition());
+    SmartDashboard.putNumber("climber motor encoder", climberMotor.get());
```
Comment (AngelaKimmy) [inline src/main/java/frc/robot/subsystems/Climber.java]: Does this string reflect the type of data we are displaying?

**Review by TaylerUva — APPROVED** [review]

**Review by TaylerUva — APPROVED** [review]

**Review by TaylerUva — APPROVED** [review]

**Pull request merged**